igor-web/src/main/groovy/com/netflix/spinnaker/igor/scm/KeelManifestsController.groovy

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.igor.scm
+
+import com.fasterxml.jackson.databind.DeserializationFeature
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory
+import com.netflix.spinnaker.igor.scm.bitbucket.client.BitBucketMaster
+import com.netflix.spinnaker.igor.scm.github.client.GitHubMaster
+import com.netflix.spinnaker.igor.scm.gitlab.client.GitLabMaster
+import com.netflix.spinnaker.igor.scm.stash.client.StashMaster
+import groovy.util.logging.Slf4j
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RequestParam
+import org.springframework.web.bind.annotation.RestController
+
+/**
+ * Exposes APIs to retrieve keel declarative manifests from source control repos.
+ */
+@RestController
+@Slf4j
+@RequestMapping("/keel")
```

Comment:
I think this is the wrong request mapping. Even though these endpoints are for keel, they should be descriptive of what they do

Reply:
maybe `scm/manifests`?

Reply:
+1 to change the mapping, it might be useful in other cases as well

Reply:
Like I said here, I struggled with naming for this URI... https://github.com/spinnaker/orca/pull/3344#discussion_r358500249

I think `manifests` is _too_ generic. What kind of manifests? Maybe `md-manifests` then?

Reply:
Looks like we settled on `delivery-configs`, @gal-yardeni and @emjburns. Thanks for the feedback!